tutorials-examples/portal-intermediate.rst

```diff
@@ -100,15 +100,15 @@ This query also uses the ccdVisitId to join to the CcdVisit table and obtain the
 3.1. If the Results view is showing three panels including a sky image at upper left, switch to the "xy-tbl" view using the buttons at upper right.
 Use the settings icon (two gears at upper right) to open the plot parameters pop-up window, match those shown below, then click "Apply" and "Close".
 
-.. figure:: /_static/portal_tut02_step03a.png
-    :name: portal_tut02_step03a
+.. figure:: /_static/portal_tut02_step03a_GM.png
```

Comment:
General comment: I would suggest replacing the original images, i.e., re-using their names, in preference to using these "GM" names.  That will make the git history (much) clearer, and will avoid leaving obsolete images around to have to be separately deleted.

Reply:
I can see that this comment about replacing original images (instead of adding _GM images) has been incorporated, and so I'm resolving this conversation.